### PR TITLE
Major update including graphical mode, background service, API updates, Swedish support, crash fixes and more

### PR DESCRIPTION
**Pull request by paulobrien**
Changes include:

- Swapped to the vehicle data API to reduce from 3 API calls to 1 (much more responsive on load!)
- Added image view (default for touchscreen devices, switchable for all devices)
- Added the option to swap the frunk action for the port action
- Enhanced the glance view for supported devices (see below)
- Added a background service for supported devices to support the glance view, does NOT keep vehicle awake
- Enabled dynamic arc thickness based on device
- Added custom font for venu2
- Reduced the logging (as it's expensive on low power devices) and added 'debug' and 'release' loggers
- Prevented touchscreen being used when a message is being displayed
- Fixed some localisations
- Fixed some issues where errors (e.g. -101) were displayed on screen
- Fixed crash issue where car name was blank
- Added Swedish support

### CODE REVIEW

File: resources-fre/strings/strings.xml [hunk 1]
```diff
@@ -3,8 +3,10 @@
     <string id="car_chooser_title">Choisir un VIN</string>
 
     <!--  Glance view -->
-    <string id="wake" scope="glance">Wake </string>
-    <string id="vehicle" scope="glance">vehicle</string>
+    <string id="wake" scope="glance">Rréveiller </string>
```
Comment (hobbe) [inline resources-fre/strings/strings.xml]:
```suggestion
    <string id="wake" scope="glance">Réveiller </string>
```

File: resources-fre/strings/strings.xml [hunk 2]
```diff
@@ -3,8 +3,10 @@
...
+    <string id="vehicle" scope="glance">véhicule</string>
+    <string id="label_asleep" scope="background">Endormie</string>
```
Comment (hobbe) [inline resources-fre/strings/strings.xml]:
```suggestion
    <string id="label_asleep" scope="background">En veille</string>
```

File: resources-fre/strings/strings.xml [hunk 3]
```diff
@@ -3,8 +3,10 @@
...
+    <string id="label_status_unknown" scope="glance">Statut inconnu</string>
```
Comment (hobbe) [inline resources-fre/strings/strings.xml]:
```suggestion
    <string id="label_status_unknown" scope="glance">État inconnu</string>
```

File: resources-fre/strings/strings.xml [hunk 4]
```diff
@@ -45,8 +47,8 @@
     <string id="label_locked">Fermée</string>
     <string id="label_login_on_phone">Connexion sur tél.</string>
     <string id="label_oauth_error">Erreur OAuth</string>
-    <string id="label_off">Off</string>
-    <string id="label_on">On</string>
+    <string id="label_off">Désactivé</string>
+    <string id="label_on">Arrêter</string>
```
Comment (hobbe) [inline resources-fre/strings/strings.xml]:
```suggestion
    <string id="label_off">Off</string>
    <string id="label_on">On</string>
```
I guess we could keep On/Off for this case and to keep text short.

File: resources-fre/strings/strings.xml [hunk 5]
```diff
@@ -40,11 +47,12 @@
...
     <string id="label_open_frunk">Ouvrir le coffre ?</string>
-    <string id="label_requesting_data">Récup. données...</string>
+    <string id="label_requesting_data">Récup. données</string>
     <string id="label_unlock_doors">Déver. portes</string>
     <string id="label_unlocked">Ouverte</string>
-    <string id="label_waking_vehicle">Réveil du vehicule...</string>
+    <string id="label_waking_vehicle">Réveil du vehicule</string>
+    <string id="label_no_vehicles">Pas de véhicules</string>
```
Comment (hobbe) [inline resources-fre/strings/strings.xml]:
```suggestion
    <string id="label_no_vehicles">Pas de véhicule</string>
```

**Review by hobbe — CHANGES_REQUESTED** [review]

**Comment by paulobrien** [comment]
Thank you for the translation fixes @hobbe 😃

**Comment by paulobrien** [comment]
Currently in discussion with @srwalter re: how best to optimise support for non-touchscreen devices in the glance view

**Comment by srwalter** [comment]
Any update on fixing support for non-touch devices?

**Comment by paulobrien** [comment]
In progress @srwalter, hence set to WIP status for now.

**Comment by paulobrien** [comment]
Finally done and ready I think! 😃

**Pull request merged**